.github/workflows/build.yml

```diff
@@ -73,6 +73,58 @@ jobs:
           name: spin-ubuntu-latest
           path: target/release/spin
 
+  build-rust-static:
```

Comment:
Any reason we're not using a `matrix` strategy for these two, to slim down the code footprint?

Reply:
no specific reason. i can change it to use matrix strategy.

Reply:
this is now done